addons/website/views/snippets.xml

```diff
@@ -365,6 +365,74 @@
     </section>
 </template>
 
+<template id="s_company_team" name="Company Team">
+    <section class="s_company_team">
+        <div class="container">
+            <div class="row mb32">
+                <div class="col-md-12 col-xs-12">
+                    <h1 class="text-center">
+                        Meet the Executive Team
+                    </h1>
+                </div>
+            </div>
+            <div class="row">
+                <div class="col-md-6 col-xs-12 mb32">
+                    <div class="col-md-4">
```

Comment:
Mmh ok, but I think that XML structure is more important. What you could do is put the row with a `s_no_resize_cols` class and change the `margin-x` option so that the `data-selector` is `.row:not(.s_no_resize_cols) > [class*='col-md-']`.